src/OpenTelemetry.Contrib.Instrumentation.AWSLambda/CHANGELOG.md

```diff
@@ -2,23 +2,24 @@
 
 ## Unreleased
 
-* Updated the `ActivitySource` name to the assembly name:
-  `OpenTelemetry.Instrumentation.AWSLambda`
+## 2.0.0
```

Comment:
Do you want to do a stable release now? You could have another beta version (`1.1.0-beta.2`) released.

Reply:
I will check back on that question internally. Are there any formal consequences of (not) using a beta release that I should be aware of for opentelemetry-dotnet-contrib (apart from signalling that it might not work properly yet and the need to specify `--prerelease` on the command line)?

Reply:
In general, it's a good practice to have a beta version released, so that people can use it for a good amount of time and provide feedback, if any.

Since, this package never had a stable release, so it should be fine to another beta version release with these breaking changes.

Reply:
I changed it to 2.0.0-beta1 in 9acd7e1

Reply:
This package has never had a stable `1.*.*` release. Is there any particular reason to jump to `2.*.*` version without having a stable `1.*.*` version release?

Reply:
I wouldn't expect such a large API change between betas. A beta is something that I would expect to be relatively stable API-wise, but maybe that's just my interpretation?

Reply:
https://github.com/open-telemetry/opentelemetry-dotnet/blob/main/VERSIONING.md#pre-releases if we go with this, its okay to have breaking changes in a beta release.
 With rc also it is allowed, but has a much higher bar than beta.

Reply:
OK, I'm fine with 1.0.0-beta2 as well, @rypdal what do you think?